README.md

```diff
@@ -30,12 +76,73 @@ This will pull a pre-built image from Docker Hub (or use an existing image if al
 If you'd rather build the model locally you can follow the steps in the
 [model README](https://github.com/IBM/MAX-Object-Detector/#steps).
 
+#### 2. Experimenting with the API (Optional)
+
+The API server automatically generates an interactive Swagger documentation page.
+Go to `http://localhost:5000` to load it. From there you can explore the API and also create test requests.
+
+Use the `model/predict` endpoint to load a test image and get predicted labels for the image from the API.
+The coordinates of the bounding box are returned in the `detection_box` field, and contain the array of normalized
+coordinates (ranging from 0 to 1) in the form `[ymin, xmin, ymax, xmax]`.
+
+The [model assets folder](https://github.com/IBM/MAX-Object-Detector/tree/master/assets)
+contains a few images you can use to test out the API, or you can use your own.
+
+You can also test it on the command line, for example:
+
+```
+$ curl -F "image=@assets/dog-human.jpg" -XPOST http://localhost:5000/model/predict
+```
+
+You should see a JSON response like that below:
+
+```json
+{
+  "status": "ok",
+  "predictions": [
+      {
+          "label_id": "1",
+          "label": "person",
+          "probability": 0.944034993648529,
+          "detection_box": [
+              0.1242099404335022,
+              0.12507188320159912,
+              0.8423267006874084,
+              0.5974075794219971
+          ]
+      },
+      {
+          "label_id": "18",
+          "label": "dog",
+          "probability": 0.8645511865615845,
+          "detection_box": [
+              0.10447660088539124,
+              0.17799153923988342,
+              0.8422801494598389,
+              0.732001781463623
+          ]
+      }
+  ]
+}
+```
+
+You can also control the probability threshold for what objects are returned using the `threshold` argument like below:
+
+```
+$ curl -F "image=@assets/dog-human.jpg" -XPOST http://localhost:5000/model/predict?threshold=0.5
+```
+
+The optional `threshold` parameter is the minimum `probability` value for predicted labels returned by the model.
+The default value for `threshold` is `0.7`.
+
 ### Start the Web App
 
+#### Use the embedded web app
+
 The latest release of the web app is deployed with the model API above and is available at
```

Comment:
Please add a blurb how the embedded mini-app can be disabled.

Reply:
Given this is the web app repo and those instructions are in the model repo I don't see a reason to include those instructions. Why would the user want to turn off the feature they're here to use. And as for the case they plan to deploy the standalone instead, there's no harm or overhead in leaving the web app on vs disabling it.

Reply:
The reason I brought this up is this sentence in this readme: "The latest release of the web app is deployed with the model API above and is available at http://localhost:5000/app".

Reply:
That blurb is specifically to state that the web app is already available from the previous step and they don't have to run the following setup unless they want to run the latest master. So that section is actually the "default instructions" for the code pattern, thus the reason telling how to disable it doesn't make sense